README.md

```diff
@@ -1,12 +1,11 @@
-open-orchestra-media-bundle
+open-orchestra-workflow-bundle
```

Comment:
Can you use workflow-fonction-bundle for the name.

We are also going to rename the repository according to the new name